<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qtResources.qrc">
    <normaloff>:/newPrefix/ui/marta-logo.png</normaloff>:/newPrefix/ui/marta-logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {background-color: #ffffff} QLabel {color: #009fd7; font-weight: bold;}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="stationManagementButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Station Management</string>
   </property>
  </widget>
  <widget class="QPushButton" name="breezeCardManagementButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Breeze Card Management</string>
   </property>
  </widget>
  <widget class="QPushButton" name="passengerFlowReportButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Passenger Flow Report</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suspendedCardsButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Suspend Cards</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="windowModality">
    <enum>Qt::ApplicationModal</enum>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="frameLabelTop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#frameLabelTop {background-color: transparent; color: #ee9306; font-size: 30px;}</string>
   </property>
   <property name="text">
    <string>Administrator</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>stationManagementButton</tabstop>
  <tabstop>suspendedCardsButton</tabstop>
  <tabstop>breezeCardManagementButton</tabstop>
  <tabstop>passengerFlowReportButton</tabstop>
  <tabstop>logOutButton</tabstop>
 </tabstops>
 <resources>
  <include location="../qtResources.qrc"/>
 </resources>
 <connections/>
</ui>
